<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Label:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_get_data">
        <property name="text">
         <string>get training data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_predict">
        <property name="text">
         <string>start predictions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_train_model">
        <property name="text">
         <string>train model</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>co&amp;nfig</string>
    </property>
    <addaction name="actionsetWorkingDir"/>
    <addaction name="actionset_n_images"/>
    <addaction name="actionclear_all_training_data"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsetWorkingDir">
   <property name="text">
    <string>&amp;setWorkingDir</string>
   </property>
  </action>
  <action name="actionset_n_images">
   <property name="text">
    <string>set_n_images</string>
   </property>
  </action>
  <action name="actionclear_all_training_data">
   <property name="text">
    <string>clear_all_training_data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
